### DISCUSSION TITLE
MSSQL Error Connection error.

### DISCUSSION THREAD

**Discussion by benembery** [prisma/prisma#4755] [Q&A] [answered]
Hi 👋, 

I'm new to prisma so I might have missed something obvious here. I'm trying to connect to a SQL Server from MacOS and getting the following error when querying the database.

```
MissingDriverError: Wrong driver: "undefined" given. Supported drivers are: "cordova", "expo", "mariadb", "mongodb", "mssql", "mysql", "oracle", "postgres", "sqlite", "better-sqlite3", "sqljs", "react-native", "aurora-data-api", "aurora-data-api-pg".
```

I've run `prisma introspect` (it's a pre-existing database) and that successfully generated the schema. 
I've also run `prisma generate`. 
I can also successfully inspect the db using `prisma studio`

I've followed the MSSQL tutorial and looked through the connector documentation and the only thing I can see that could be causing this issue is the database is SQL Server 2016. Is this the reason?

**Comment by ryands17** [comment, marked as answer]
Hey @benembery 👋 
This doesn't look an error coming out of Prisma. Do you have any other package installed that might give this error which might be possibly coming from TypeORM?

There are some related searches also that have the same error: https://github.com/typeorm/typeorm/issues/4068.

**Reply by benembery** [reply]
You are right, it was down Next Auth also using the env variable DATABASE_URL. By default, it uses TypeORM. Thanks for taking the time to reply.

**Comment by shrugs** [comment]
chiming in for future googlers: you probably used `database:` in your next-auth config instead of `adapter:`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
